src/style.css

```diff
@@ -0,0 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Lato', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.navbar {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 10px;
+  background-color: #000;
+}
+
+.navitems,
+.logo {
+  cursor: pointer;
+}
+
+.logo {
+  width: 5%;
+  border-radius: 50%;
+  margin-left: 12rem;
+}
+
+.navlinks {
+  display: flex;
+  margin-top: 3rem;
+}
+
+.navitems,
+a {
+  margin-right: 7rem;
+  font-size: 1.8rem;
+  color: #fff;
+}
+
+a:hover {
+  color: hotpink;
+  border-bottom-style: solid;
+}
+
+.recipe-count {
+  margin-right: 9rem;
+}
+
+.mainCont {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 10px;
+  grid-auto-rows: minmax(100px, auto);
+  background-color: #d0d0e2;
+  padding: 30px;
+  width: 80%;
+  margin: auto;
+}
+
+.meal-desc {
+  display: flex;
+  flex-direction: column;
+  color: #fff;
+  font-weight: 900;
+  font-size: 1.6rem;
+  text-align: right;
+}
+
+.name-like {
+  display: flex;
+  justify-content: space-between;
+  padding: 18px 50px;
+}
+
+.comment-btn,
+.reserve-btn {
+  display: flex;
+  width: 350px;
+  background-color: rgb(213, 182, 182);
+  justify-content: center;
+  border-radius: 15px;
+  padding: 8px 0;
+  margin: 12px 0;
+  text-transform: uppercase;
+  font-size: 1.3rem;
+  cursor: pointer;
+}
+
+.comment-btn:hover,
+.reserve-btn:hover {
+  color: white;
+  background-color: burlywood;
+}
+
+.button:hover {
+  cursor: pointer;
+  color: #fff;
+}
+
+.btn-div {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  margin: 10px 180px;
+}
+
+.likes-count {
+  color: black;
+  font-size: 1.8rem;
+  align-items: flex-end;
+  margin-right: 50px;
+}
+
+.fa-heart {
+  color: #fff;
+  font-size: 30px;
+}
+
+.fa-heart:hover {
+  color: black;
+  cursor: pointer;
+}
+
+.meal-photo {
+  background-color: rgb(151, 135, 113);
+  border-radius: 30px;
+  overflow: hidden;
+  flex-wrap: wrap;
+}
+
+.img {
+  width: 200px;
+  height: 300px;
+  object-fit: contain;
+}
```

Comment:
- [ ] the images "fall" off the page when opening the application as you can see in the below screenshot. please fix this.
![afbeelding](https://user-images.githubusercontent.com/39240820/219809257-6c1fe98a-75e7-402f-812e-f3864817576a.png)